cmd/create/accountroles/cmd.go

```diff
@@ -363,7 +363,18 @@ func createRoles(reporter *rprtr.Object, awsClient aws.Client,
 		filename = fmt.Sprintf("sts_%s_permission_policy.json", file)
 		path = fmt.Sprintf("templates/policies/%s", filename)
 
-		policy, err = aws.ReadPolicyDocument(path)
+		// For the installer permission policy scope the iam:PassRole permission
+		// to the controlplane ARN role only.
+		// The cluster itslef will provision worker nodes see machine-api-operator
+		// permission.
+		if strings.ToLower(role.Name) == aws.InstallerAccountRole {
+			policy, err = aws.ReadPolicyDocument(path, map[string]string{
+				"aws_account_id":   accountID,
+				"controlplane_arn": aws.GetRoleName(prefix, aws.AccountRoles[aws.ControlPlaneAccountRole].Name),
+			})
+		} else {
+			policy, err = aws.ReadPolicyDocument(path)
+		}
```

Comment:
You probably don't need to wrap this in a conditional. The templating should replace only if the format string exists.